src/Admin/ErrorLogAdmin.php

```diff
@@ -57,10 +52,7 @@ protected function configureShowFields(ShowMapper $show)
             ->add('message');
     }
 
-    /**
-     * {@inheritDoc}
-     */
-    protected function configureRoutes(RouteCollection $collection)
+    protected function configureRoutes(RouteCollectionInterface $collection): void
```

Comment:
Wel jammer dat er vanwege deze change een compleet nieuwe major versie nodig is... 😢